diff --git a/layers/+lang/latex/packages.el b/layers/+lang/latex/packages.el
index cdd02e634..c0b00a156 100644
--- a/layers/+lang/latex/packages.el
+++ b/layers/+lang/latex/packages.el
@@ -121,6 +121,7 @@
         "."   'LaTeX-mark-environment  ;; C-c .
         "ii"   'LaTeX-insert-item       ;; C-c C-j
         "s"   'LaTeX-section           ;; C-c C-s
+        "fb"  'LaTeX-fill-buffer       ;; C-c C-q C-e
         "fe"  'LaTeX-fill-environment  ;; C-c C-q C-e
         "fp"  'LaTeX-fill-paragraph    ;; C-c C-q C-p
         "fr"  'LaTeX-fill-region       ;; C-c C-q C-r
